chirp/drivers/ft1d.py

```diff
@@ -1147,31 +1147,32 @@ def _get_aprs_beacons(self):
 
         for index in range(0, 60):
             # There is probably a more pythonesque way to do this
-            if int(aprs_meta[index].sender_callsign[0]) != 255:
+            scs = int(aprs_meta[index].sender_callsign[0])
+            dcs = int(aprs_beacon[index].dst_callsign[0])
+            if scs != 0xFF and scs != 0:
                 callsign = str(aprs_meta[index].sender_callsign).rstrip("\xFF")
-                # LOG.debug("Callsign %s %s" % (callsign, list(callsign)))
                 val = RadioSettingValueString(0, 9, callsign)
                 val.set_mutable(False)
                 rs = RadioSetting(
                     "aprs_beacon.src_callsign%d" % index,
                     "SRC Callsign %d" % index, val)
                 menu.append(rs)
 
-            if int(aprs_beacon[index].dst_callsign[0]) != 255:
-                val = RadioSettingValueString(
-                        0, 9,
-                        str(aprs_beacon[index].dst_callsign).rstrip("\xFF"))
-                val.set_mutable(False)
-                rs = RadioSetting(
-                        "aprs_beacon.dst_callsign%d" % index,
-                        "DST Callsign %d" % index, val)
-                menu.append(rs)
+                if dcs != 0xFF and dcs != 0:
+                    val = RadioSettingValueString(
+                            0, 9,
+                            str(aprs_beacon[index].dst_callsign).rstrip("\xFF"))
+                    val.set_mutable(False)
+                    rs = RadioSetting(
+                            "aprs_beacon.dst_callsign%d" % index,
+                            "DST Callsign %d" % index, val)
+                    menu.append(rs)
 
-            if int(aprs_meta[index].sender_callsign[0]) != 255:
                 date = "%02d/%02d/%02d" % (
                     aprs_meta[index].date[0],
                     aprs_meta[index].date[1],
                     aprs_meta[index].date[2])
+                LOG.warn("\nDate=%s", date)
```

Comment:
This isn't really a good use for being a warning, as far as I can tell. I assume this is just debug right? Also please don't put newlines in log messages unless absolutely necessary.